portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div id="niceback">
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Michael's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<div id="content">
+      <h1><p3 style="color:white">Michael Peters's Portfolio </p3></h1>
+      <center><img src="/images/IMG_3549.jpg" width="120" height="190"></center>
+      
+      
+      
+
+<div id="greeting-container"></div>
     </div>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>About Me</p4>
+    </div id="Me">
+    <p1 style="color:white">My name is Michael Peters. I was born and raised in Southern California. 
```

Comment:
Move this style into the css file please (and all instances below). We want to try and stay away from styling inside the html files because it can cause unexpected behavior due to CSS precedence rules of inline styling overriding CSS file styling.